### ISSUE TITLE
车架号检测如何配置更合适

### ISSUE THREAD

**Issue by soulocy** [PaddlePaddle/PaddleOCR#9130]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

目前对汽车的铭牌、挡风玻璃、行驶证进行车架号文本检测，有16万样本量，训练完成后检测结果不理想。
1、请问有什么好的方法提升检测的准确率？
2、配置文件中 MakeShrinkMap的min_text_size我需要调整到17吗？车架号是由17位的数字字母组合。

![NT}D6%VRBAI7$T9$}I6G{G4](https://user-images.githubusercontent.com/125716511/219843809-5919ec0d-575a-4d04-be42-f88e31b4d4b7.png)

best metric, hmean: 0.41395135383203296, is_float16: False, precision: 0.7204472843450479, recall: 0.2904056664520283

样例
![1a497663693ace84](https://user-images.githubusercontent.com/125716511/219830039-0da44045-9729-4a56-9f9b-ee76f2fef43c.jpg)

测试结果
![G{KGXG4N0H Z($O}I((DKA8](https://user-images.githubusercontent.com/125716511/219843723-1431f3ad-1655-4fa2-9d55-7f3fa93da064.png)

3、标注时非0度的图片是否需要旋转再标注？训练样本中有30%图片是非0度的
![M_DA3LKOH@EL}@T}(N)(TI9](https://user-images.githubusercontent.com/125716511/219849364-51334827-ba51-4330-acf0-5bc656fecbc8.png)

配置文件
#det_r50_vd_db
Global:
  use_gpu: true
  epoch_num: 100
  log_smooth_window: 100
  print_batch_step: 200
  save_model_dir: ./train_data/vin/model_det/
  #设置模型保存间隔 epoch_num
  save_epoch_step: 10
  #设置模型评估间隔
  eval_batch_step: 2000
  cal_metric_during_train: False
  pretrained_model:
  checkpoints:
  save_inference_dir:
  use_visualdl: False
  infer_img: ./train_data/vin/test_det/
  save_res_path: ./train_data/vin/pre_vin_detdb.txt

Architecture:
  model_type: det
  algorithm: DB
  Transform:
  Backbone:
    name: ResNet_vd
    layers: 50
  Neck:
    name: DBFPN
    out_channels: 256
  Head:
    name: DBHead
    k: 50

Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    learning_rate: 0.001
  regularizer:
    name: 'L2'
    factor: 0

PostProcess:
  name: DBPostProcess
  thresh: 0.3
  box_thresh: 0.7
  max_candidates: 1000
  unclip_ratio: 1.5

Metric:
  #指标评估方法名称
  name: DetMetric
  #用于选取最优模型
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/
    label_file_list:
      - ./train_data/vin/imgs/2022/train_chk.txt
    #ratio_list: [1.0]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - IaaAugment:
          augmenter_args:
            - { 'type': Fliplr, 'args': { 'p': 0.5 } }
            - { 'type': Affine, 'args': { 'rotate': [-10, 10] } }
            - { 'type': Resize, 'args': { 'size': [0.5, 3] } }
      - EastRandomCropData:
          size: [640, 640]
          max_tries: 50
          keep_ratio: true
      - MakeBorderMap:
          shrink_ratio: 0.4
          thresh_min: 0.3
          thresh_max: 0.7
      - MakeShrinkMap:
          shrink_ratio: 0.4
          min_text_size: 8
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'threshold_map', 'threshold_mask', 'shrink_map', 'shrink_mask'] # the order of the dataloader list
  loader:
    shuffle: True
    drop_last: False
    batch_size_per_card: 8
    num_workers: 4

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/
    label_file_list:
      - ./train_data/vin/imgs/2022/test_chk.txt
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - DetResizeForTest:
          image_shape: [736, 1280]
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'shape', 'polys', 'ignore_tags']
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1 # must be 1
    num_workers: 8
